### PR TITLE
Release `v3.3.1`

### PR DESCRIPTION
**Pull request by andschwa**
Automated PR for new release!

### CODE REVIEW

**Review by SydneyhSmith — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # PowerShell Editor Services Release History
 
+## v3.3.1
+### Wednesday, April 20, 2022
+
+- 🐛 👷 [PowerShellEditorServices #1761](https://github.com/PowerShell/PowerShellEditorServices/pull/1766) - Bump `net461` to `net462` due to upcoming end of support.
+- 🐛 💎 [vscode-powershell #3928](https://github.com/PowerShell/PowerShellEditorServices/pull/1764) - Fix formatting handlers and PSScriptAnalyzer loading.
```
Comment (SydneyhSmith) [inline CHANGELOG.md:7]: is there supposed to be a vscode one in here?

Comment (andschwa) [reply CHANGELOG.md:7]: The script is built to put the bug number (if referenced in the PR) but link to the fix (the PR itself), so this is expected but looks funny.

Comment (SydneyhSmith) [reply CHANGELOG.md:7]: Ah I see that now, cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool

Comment (SydneyhSmith) [reply CHANGELOG.md:7]: GitHub suggested the extra cools

Comment (andschwa) [reply CHANGELOG.md:7]: 😆

**Pull request merged**